crates/flux-refineck/src/primops.rs

```diff
@@ -171,14 +173,14 @@ fn mk_mul_rules(check_overflow: bool) -> RuleMatcher<2> {
             requires E::le(a * b, E::uint_max(uint_ty)) => ConstrReason::Overflow
             if let &BaseTy::Uint(uint_ty) = T
 
-            fn(a: T, b: T) -> T
+            fn(a: T, b: T) -> T // this is underspecified
```

Comment:
Please remove these comments.

```suggestion
            fn(a: T, b: T) -> T
```